docs/spec/spec.md

```diff
@@ -76,15 +76,15 @@ _Describes a linkage between a Channel and a Targetable and/or Sinkable._
 
 #### Spec
 
-| Field              | Type           | Description                                                                  | Constraints        |
-| ------------------ | -------------- | ---------------------------------------------------------------------------- | ------------------ |
-| from\*             | ObjectRef      | The originating _Subscribable_ for the link.                                 | Must be a Channel. |
-| call<sup>1</sup>   | EndpointSpec   | Optional processing on the event. The result of call will be sent to result. |                    |
-| result<sup>1</sup> | ResultStrategy | The continuation for the link.                                               |                    |
+| Field                  | Type           | Description                                                                         | Constraints        |
+| ---------------------- | -------------- | ----------------------------------------------------------------------------------- | ------------------ |
+| channel\*              | ObjectRef      | The originating _Subscribable_ for the link.                                        | Must be a Channel. |
```

Comment:
I'm not a big fan of changing this to `channel` instead of `from`. I'm concerned that this generates unnecessary coupling between an attractive user-facing name ("Subscription") and a single core messaging implementation ([PubSub Topic](https://www.enterpriseintegrationpatterns.com/patterns/messaging/PublishSubscribeChannel.html), aka Channel). For example, this definition would make it impossible to subscribe to a [Messaging Bridge](https://www.enterpriseintegrationpatterns.com/patterns/messaging/MessagingBridge.html).

Talking it over with @grantr, I think I can be okay with this due to the following mitigations:

1. We're going to revisit the object model in 3-6 months and see what we need to adjust. At that time, we may end up with new primitives and it's okay for Subscription to be tightly bound to Channel.
2. We can add a generic `from` field later and do a version upgrade/one-of field behavior between `channel` and `from`.

Reply:
A bridge connects two channels, typically in disparate messaging systems. The channel still acts as the interface in the local messaging system.